hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java

```diff
@@ -1911,14 +1882,6 @@ protected Subject getSubject() {
     }
   }
 
-  private void logPrivilegedAction(Subject subject, Object action) {
-    if (LOG.isDebugEnabled()) {
-      // would be nice if action included a descriptive toString()
-      String where = new Throwable().getStackTrace()[2].toString();
```

Comment:
I would fix this method instead of replacing.

Reply:
This hardly seems like it should merit its own method.  It's trivial to do in-line and it seems like a bad pattern to have... every class that wants this functionality would need its own copy of this routine.  I don't think we want that.  It would be better in a utility class, that everyone can share, but I think that is overkill.